src/router/index.js

```diff
@@ -30,6 +31,11 @@ const router = new VueRouter({
       name: 'about-page',
       component: AboutPage,
     },
+    {
+      path: '/help',
```

Comment:
maybe call this `search-help` or something like that in case we have a more generic `help` page later?